xclim/indices/run_length.py

```diff
@@ -12,7 +12,6 @@
 from warnings import warn
 
 import numpy as np
-import xarray
```

Comment:
If at any point there are docstring examples introduced here, we'll need to resort to using `import xarray`. This is because the `xr` used in the docstrings is a reference to `import xcim.testing as xr` injected in the doctest namespace. Not an issue here right now, but an FYI.

Reply:
Got it, thanks.